src/tests/ftest/scripts/setup_nodes.sh

```diff
@@ -71,6 +71,8 @@ rm -f /tmp/test.cov
 if [ -f /usr/lib/daos/TESTING/ftest/test.cov ]; then
     cp /usr/lib/daos/TESTING/ftest/test.cov /tmp
     chmod 777 /tmp/test.cov
+##DH++
```

Comment:
What is this?

Reply:
Not sure if this is the correct location to export env variable.
This is just a comment, not for commit yet.